scripts/2-hyperkube-release.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+set -euo pipefail
+
+set -x
+
+# Required Settings
+echo "Release Tag: ${RELEASE_TAG}"
+echo "Push Image: ${PUSH_IMAGE}"
+# TODO(aaron): remove this when tests no longer need to be containerized
+echo "GO_VERSION: ${GO_VERSION}"
+
+git clone https://github.com/coreos/kubernetes
+
+cd kubernetes
+KUBE_ROOT=$(git rev-parse --show-toplevel)
+
+git show-ref --verify refs/tags/${RELEASE_TAG}
+git checkout ${RELEASE_TAG}
+
+# NOTE(pb): until make release works, containerize make test
+# and run that then run quick-release
+RKT_OPTS=$(echo \
+  "--volume=k8s,kind=host,source=${KUBE_ROOT} " \
+  "--mount volume=k8s,target=/go/src/k8s.io/kubernetes" \
+  "--volume selinux,kind=host,source=/sys/fs/selinux,readOnly=true" \
+  "--mount volume=selinux,target=/sys/fs/selinux" \
+  "--stage1-from-dir=stage1-fly.aci" )
+
+MAKE_TESTS=$(echo \
+  "cd /go/src/k8s.io/kubernetes &&" \
+  "apt-get update &&" \
+  "apt-get install -y rsync &&" \
+  "KUBE_TIMEOUT=\"--timeout 600s\" make test")
+sudo rkt run --insecure-options=image ${RKT_OPTS} docker://golang:${GO_VERSION} --exec /bin/bash -- -c "${MAKE_TESTS}"
+sudo chown -R jenkins:jenkins ./
```

Comment:
Hm, I'm surprised this works.

Reply:
I need to delete the 2- script... It isn't used in this context.